docs/source/devs/env.rst

```diff
@@ -50,7 +71,12 @@ The tutorials are run inside a specialized Docker container
 (not the development container).
 To test whether the Binder container is working properly, run the
 ``binder/test.sh`` script from the repository root
-(i.e., not from inside the development container).
+(i.e., not from inside the development container):
+
+.. code:: shell
+
+   $ source binder/test.sh
```

Comment:
```suggestion
   $ binder/test.sh
```
should suffice - can you double-check this?

Reply:
The correct permissions were missing:

``` shell
bash-3.2$ binder/test.sh
bash: binder/test.sh: Permission denied
bash-3.2$ ./binder/test.sh
bash: ./binder/test.sh: Permission denied
```

I've pushed a commit to fix this.

Reply:
Awesome, thanks - I'm usually developing in WSL, which doesn't give me good control over file permissions.